src/main/java/thefarm/orchards/apple/model/Apple.java

```diff
@@ -0,0 +1,88 @@
+package thefarm.orchards.apple.model;
+
+public class Apple {
+    private String stem;
+    private String blossom;
+    private String peeling;
+    private String juice;
+    private String color;
+
+    public Apple(String stem, String blossom, String peeling, String juice, String color) {
+        this.stem = stem;
+        this.blossom = blossom;
+        this.peeling = peeling;
+        this.juice = juice;
+        this.color = color;
+    }
+
+    public String getPeeling() {
+        return peeling;
+    }
+
+    public void setPeeling(String peeling) {
+        this.peeling = peeling;
+    }
+
+    public String getJuice() {
+        return juice;
+    }
+
+    public void setJuice(String juice) {
+        this.juice = juice;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getSize() {
+        return size;
+    }
+
+    public void setSize(String size) {
+        this.size = size;
+    }
+
+    public String getTexture() {
+        return texture;
+    }
+
+    public void setTexture(String texture) {
+        this.texture = texture;
+    }
+
+    public String getDamage() {
+        return damage;
+    }
+
+    public void setDamage(String damage) {
+        this.damage = damage;
+    }
+
+    private String size;
+    private String texture;
+    private String damage;
```

Comment:
How would you describe damage? Does it matter what the damage is? Or should it be a level of damage (none, moderate, heavy)? Or perhaps just a simple isDamaged with a boolean?

Reply:
This would be more of a list of potential damages: Rot, bruise, bugs, worms...

Reply:
The question here is a little more complex.

Would you say that a damaged apple contains and describes its damage? Or does the damage apply to an apple and describe the apple?

There might be an Enum further into the app which can be used universally by more than one type of fruit, perhaps. There could be a list, perhaps called ``rotten`` which could contain multiple Apple objects, some Oranges, a Peach or two, and perhaps a Watermelon?